### DISCUSSION TITLE
Change order of lists

### DISCUSSION THREAD

**Discussion by Hona** [community/community#8629] [Other Feature Feedback, Questions, & Ideas]
Related: #8614 

I would like to be able to change the order of lists, for example:

I would like both the `C# / .NET ...` lists to be next to each other. 

![image](https://user-images.githubusercontent.com/10430890/145491245-38129c87-a7b5-4da6-a029-011e30aaf398.png)

**Comment by brianlovin** [comment]
Thanks for the suggestion, agreed. We've had a few other people request the same, so I'll add your +1 to our tracking issue 😊
